camel/toolkits/hybrid_browser_toolkit/browser_session.py

```
@@ -65,26 +68,286 @@ def __new__(
+                script_content = f.read()
+
+            if not script_content.strip():
+                raise ValueError(f"Stealth script is empty: {script_path}")
+
+            logger.debug(
+                f"Loaded stealth script ({len(script_content)} chars)"
+            )
+            return script_content
+        except FileNotFoundError:
+            logger.error(f"Stealth script not found: {script_path}")
+            raise FileNotFoundError(f"Stealth script not found: {script_path}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)